README.md

```diff
@@ -11,30 +11,386 @@ For more detailed information, visit the [Timeseer.AI website](https://www.times
 - Ansible 2.9 or higher.
 - docker installed on the target machine.
 - `community.docker` Ansible collection.
+- `containers.podman collection` Ansible collection if you are using podman
 
-## Example Playbook
+## Role Implementation Examples
+
+The Timeseer Ansible collection includes four roles.
+
+The `timeseer` role is primarily designed for deploying just the Docker container and is ideal for a limited proof of concept (POC) involving 1-2 users.
+
+This setup does not require authentication or a web server and can be run locally.
+
```

Comment:
```suggestion
```

Blank lines start a new paragraph. It is allowed to have multiple sentences in a paragraph. Each sentence in a paragraph should be on its own line in Markdown or asciidoc.